pkg/dao/rpms.go

```diff
@@ -777,11 +784,18 @@ func (r *rpmDaoImpl) ListTemplateErrata(ctx context.Context, orgId string, templ
 	}
 
 	for _, pkg := range pkgs {
+		issuedDate := ""
 		updatedDate := ""
 		CVEs := []string{}
 		if pkg.UpdatedDate != nil {
-			updatedDate = *pkg.UpdatedDate
+			if t, err := time.Parse(time.DateTime, *pkg.UpdatedDate); err == nil {
+				updatedDate = t.UTC().Format(time.RFC3339)
+			}
+		}
+		if t, err := time.Parse(time.DateTime, pkg.IssuedDate); err == nil {
+			issuedDate = t.UTC().Format(time.RFC3339)
```

Comment:
we might could move this UTC conversion into tang itself for both errata list methods, but i think this is fine for now